scss/detail.scss

```diff
@@ -1,3 +1,24 @@
+.card {
+  display: flex;
+  flex-flow: row wrap;
+  height: 100%;
+  max-height: 416px;
+  max-width: 900px;
+  padding: 24px;
```

Comment:
Try to round margins/paddings, example: `25px` would be nice :)

Reply:
I think Julian is using 4px like base so all values are going to be a multiply of 4